<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BattleChipView</class>
 <widget class="QDialog" name="BattleChipView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BattleChip Gate</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="5" column="1">
    <widget class="QCheckBox" name="inserted">
     <property name="text">
      <string>Inserted</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Chip ID</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Chip name</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="chipId">
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="chipName"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Gate type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="gateProgress">
     <property name="text">
      <string>Progress &amp;Gate</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">gate</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QRadioButton" name="gateBattleChip">
     <property name="text">
      <string>Ba&amp;ttleChip Gate</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">gate</string>
     </attribute>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QRadioButton" name="gateBeastLink">
     <property name="text">
      <string>Beast &amp;Link Gate</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">gate</string>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="gate"/>
 </buttongroups>
</ui>
